packages/odyssey/src/scss/base/_iconography.scss

```diff
@@ -4,4 +4,16 @@
   width: 1em;
   height: 1em;
   vertical-align: middle;
+
```

Comment:
@arnoldsandoval-okta these contextual styles should work everywhere we're using Icon today:

If it's wrapped in a container like `.ods-toast--icon` or displayed in a single cell, there's no margin as desired. Otherwise, we now have a standard fallback margin for all sibling Icons.

Individual components may, of course, override this fallback.

Reply:
^ No longer applies.